Automattic/wp-calypso
# Fix | Jetpack "Backup in Progress" screen flashes another screen before it

Fixes 1164141197617539-as-1200760980560241/f

When there is a back up in progress, there flashes another screen related to previous backups etc. before the actual backup in progress screen. The reason for that being the rewind backups are not loaded when the activity log is loaded, so the backup in progress UI which depends upon the backups gets delayed until backups are fetched/initialized.

#### Changes proposed in this Pull Request

* Display the loading placeholder until the backups are loaded/initialized.

#### Testing instructions

* Boot up this PR and run `yarn start`
* Fire up a JN site and add a backup plan for it.
* Goto `http://calypso.localhost:3000/backup/:site`
* Goto Rewind Debugger for the site and Queue a backup
* Refresh `http://calypso.localhost:3000/backup/:site` above
* Check that you don't see any screen before Backup in progress UI


Before
![demo2](https://user-images.githubusercontent.com/18226415/151948184-6ae88b5e-1874-4dc3-a882-8b6bc71562be.gif)
After
![demo](https://user-images.githubusercontent.com/18226415/151948156-3c2f459d-63c6-4c09-bb54-d63bc92da2a9.gif)
